compose/service/chart.go

```diff
@@ -249,6 +270,10 @@ func (svc chart) updater(ctx context.Context, namespaceID, chartID uint64, actio
 			}
 		}
 
+		if err = updateTranslations(ctx, svc.ac, svc.locale, c.EncodeTranslations()...); err != nil {
```

Comment:
Updater should assign a reportID in case it is not provided; same as with create but in an if that checks if it should